splice_machine/src/test/java/com/splicemachine/derby/impl/sql/pyprocedure/PyStoredProcedureResultSetFactoryIT.java

```diff
@@ -0,0 +1,224 @@
+    static final String DROP_JPROC = String.format("DROP PROCEDURE %s.JPROC_TYPE_UNIT_TEST",SCHEMA_NAME);
+    static final String CALL_JPROC = String.format("CALL %s.JPROC_TYPE_UNIT_TEST()",SCHEMA_NAME);
+
+    public static Connection getConnection() throws SQLException
+    {
+        Connection conn = DriverManager.getConnection(DB_URL, USER, PASSWORD);
+        return conn;
+    }
+
+    @BeforeClass
+    public static void setUpClass() throws Exception
+    {
+        // Create the TEST_TABLE and populate it
+        spliceClassWatcher.executeUpdate(DROP_TEST_TABLE_IF_EXIST);
+        spliceClassWatcher.executeUpdate(CREATE_TEST_TABLE);
+        spliceClassWatcher.executeUpdate(INSERT_TEST_TABLE_WITH_NULL);
+        spliceClassWatcher.executeUpdate(INSERT_TEST_TABLE_WITH_VALUE);
+        // Create the Python Stored Procedure
+        spliceClassWatcher.executeUpdate(CREATE_PYPROC);
+        spliceClassWatcher.executeUpdate(CREATE_JPROC);
+    }
+
+    @AfterClass
+    public static void tearDownClass() throws Exception
+    {
+        spliceClassWatcher.executeUpdate(DROP_TEST_TABLE_IF_EXIST);
+        spliceClassWatcher.executeUpdate(DROP_JPROC);
+        spliceClassWatcher.executeUpdate(DROP_PYPROC);
+    }
+
+    @Test
+    public void testStoredProcedure() throws Exception
+    {
+
+        ResultSet pyResultSet = methodWatcher.executeQuery(CALL_PYPROC);
+        Queue<Object[]> pyRows = new LinkedList<>();
+
+        while(pyResultSet.next()){
+            Object[] pyRow = new Object[COL_NUM];
+            pyRow[0] = pyResultSet.getLong(1);          // BIGINT
+            pyRow[1] = pyResultSet.getBoolean(2);       // BOOLEAN
+            pyRow[2] = pyResultSet.getString(3);        // CHAR
+            pyRow[3] = pyResultSet.getDate(4);          // DATE
+            pyRow[4] = pyResultSet.getBigDecimal(5);    // DECIMAL
+            pyRow[5] = pyResultSet.getDouble(6);        // DOUBLE
+            pyRow[6] = pyResultSet.getDouble(7);        // FLOAT
+            pyRow[7] = pyResultSet.getInt(8);           // INTEGER
+            pyRow[8] = pyResultSet.getBigDecimal(9);    // NUMERIC
+            pyRow[9] = pyResultSet.getFloat(10);        // REAL
+            pyRow[10] = pyResultSet.getShort(11);       // SMALLINT
+            pyRow[11] = pyResultSet.getTime(12);        // TIME
+            pyRow[12] = pyResultSet.getTimestamp(13);   // TIME_STAMP
+            pyRow[13] = pyResultSet.getString(14);      // VARCHAR
+
+            pyRow[14] = getStr(pyResultSet.getClob(15));// CLOB
+
+            pyRow[15] = getStr(pyResultSet.getClob(16));// TEXT
+            pyRows.add(pyRow);
+        }
+        pyResultSet.close();
+
+        ResultSet javaResultSet = methodWatcher.executeQuery(CALL_JPROC);
```

Comment:
This call fails in master in the mem profile. Fix or disable, make sure mem ITs pass.

Reply:
Thanks Oleg, I pushed a change to fix men platform.